### DISCUSSION TITLE
How can I use copilot on developer pc with ssl inspection?

### DISCUSSION THREAD

**Discussion by vasily-v-zaytsev** [community/community#37468] [Copilot Conversations]
Hello.
How can I use copilot on developer pc with ssl inspection?

**Comment by iamcjwilliams** [comment]
@vasily-v-zaytsev Can you provide a bit more detail? What error are you receiving?

**Reply by vasily-v-zaytsev** [reply]
Hello, [Error] An error occurred for 'RunOperation:RunOperation': StreamJsonRpc.RemoteInvocationException: Request signInInitiate failed with message: self signed certificate in certificate chain
   at StreamJsonRpc.JsonRpc.<InvokeCoreAsync>d__143`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitHub.Copilot.CopilotClient.<RunOperation>d__14`1.MoveNext()
NotSignedIn

**Reply by iamcjwilliams** [reply]
I spoke with the copilot team and currently, support for self-signed certificates is a limitation in our current proxy support. The copilot team is aiming to implement the functionality in the future. We are unable to share a specific timeline for adding self-signed certificate support at this point in time.

**Reply by vasily-v-zaytsev** [reply]
Hello, have I some workaround options for copilot on developer pc with ssl inspection (maybe live share)?
I don't have such problem on another developer pc, but on it I don't have all server environment.
